### DISCUSSION TITLE
Issue posting to WiX toolset wix-devs & wix-users

### DISCUSSION THREAD

**Discussion by MarkStega** [wixtoolset/issues#6655] [Questions] [answered]
Something appears to be blocking the posting messages from mark@optimiumhealth.com

I sent an email to both wix-devs and wix-users. I do not get a bounce back but the message does not appear.

I tried a non-verified email (qsi@stega.us) send to wix-devs and received a return email saying that (as expected) that it was a non verified email address and gave instructions on how to subscribe.

To test, I registered qsi@stega.us on wix-devs. I sent an email to wix-devs and no posting appeared. I have the option set to send an email of my own postings and nothing appears there either.

I am receiving messages from wix-users from other users so it appears that my account is properly setup.

I sent this message to wix-users-owner@lists.wixtoolset.org and didn't receive a reply.

How would you recommend proceeding?

**Comment by robmen** [comment, marked as answer]
Start the discussion here. 😄 

In a new thread, of course.

**Reply by MarkStega** [reply]
I'll do that, but it is annoying that the message list isn't working.

**Reply by robmen** [reply]
Yeah, I don't have much better guidance for the message list. It's a matter of figuring out what it is about your mail configuration is preventing the mailing list from being happy. You're welcome to keep plugging away at it, I just don't have any new ideas.

What you described during the WiX Online Meeting sounded like very similar to ours at FireGiant except we're able to communicate with the mailing lists.

**Reply by MarkStega** [reply]
The issue is that messages to the lists have to be in plain text format
